css/product-item.css

```diff
@@ -0,0 +1,109 @@
+.product-item-body {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    padding: 8rem;
+}
+
+.section-left {
+    width: 500px;
+    height: 500px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+.product-image {
+    width: 100%;
+    max-width: 350px;
+}
```

Comment:
Here also, you can use rem units.

Reply:
Will keep these suggestions in mind going forward. Thank you!